### PR TITLE
[Feat] : Implemented Playlist

### PR DESCRIPTION
**Pull request by parthpandyappp**
- Users can create a playlist.
- Users can add videos to the playlist.
- Users can remove videos from the playlist.
- Users can delete the playlist.

### CODE REVIEW

**Review by Sharmaryan — APPROVED** [review]

File: src/helper-functions/category-helpers.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+    else if (category === "Movies") {
+        return [...videos].filter((item) => item.categoryName === "Movies");
+    }
+    else if (category === "Current Affairs") {
+        return [...videos].filter((item) => item.categoryName === "Current Affairs");
+    }
+    else {
+        return videos;
+    }
```
Comment (sonishreyas) [inline src/helper-functions/category-helpers.js:19]: you can have a switch case here

**Review by sonishreyas — APPROVED** [review]
Code looks good, like the folder structure. I had one suggestion which you can implement later, rest looks good

File: src/helper-functions/category-helpers.js (hunk 1 above)
Comment (parthpandyappp) [reply src/helper-functions/category-helpers.js:19]: Okay, resolving it in next release :+1:

**Comment by parthpandyappp** [comment]
Thanks @sonishreyas and @Sharmaryan for the PR review :+1:

**Pull request merged**